docs/ManualClusterSetup.md

```diff
@@ -4,32 +4,39 @@ This document describes requirements and best practices for setting up a cluster
 
 ## Requirements
 
-2 servers (bare metal or VMs) with a modern 64-bit Linux OS, and Layer-2 network (Ethernet) connectivity between them.
+Two or more servers (bare metal or VMs) with a modern 64-bit Linux OS, each with a unique hostname.
 
 They must have the following software installed.
  * Docker v1.8 or greater: [Experimental Docker](https://experimental.docker.com)
- * etcd installed and available on each node: [etcd Documentation](https://coreos.com/etcd/docs/2.0.8/)
+ * etcd installed and available on each node: [etcd Documentation](https://coreos.com/etcd/docs/latest/)
+ * Consul installed and available on each node: [Consul Documentation](https://www.consul.io/docs/index.html)
+   * Consul is required for Docker Networking.  etcd support is planned, but currently broken.
+   * Do not run Consul in a Docker container; it must be available when the Docker daemon starts.
  * `ipset`, `iptables`, and `ip6tables` kernel modules.
 
-## Best Practices
+## Getting a Docker 1.8 Experimental Binary
 
-### CoreOS
-A prepackaged version of CoreOS with the required version of Docker is not yet available.  Calico uses the [libnetwork plugin](https://github.com/docker/libnetwork) which is currently only available in Experimental Docker v1.8.
+An experimental Docker binary is usually not available via your Linux distribution's package repos, so you will need to install it directly.
 
-If CoreOS is your distribution, we recommend viewing an earlier version of our [Manual Cluster Setup](https://github.com/Metaswitch/calico-docker/blob/powerstrip-archive/docs/ManualClusterSetup.md) which uses the Powerstrip plugin.
+We recommend that you download your Docker binary from [the release page](https://github.com/Metaswitch/calico-docker/releases) of the Calico version you are using, rather than experimental.docker.com, since we will have had a chance to check it functions with Calico.
 
-### Ubuntu
-The Ubuntu-managed Docker package is out of date and will not work with `calicoctl`.  We recommend you follow the instructions in the comments of the [Experimental Docker Website](https://experimental.docker.com) to get an up-to-date Docker install.
+For example, as root: 
 
-### Final checks
+    wget https://github.com/Metaswitch/calico-docker/releases/download/v0.5.2/docker
+    chmod +x docker
+    cp docker `which docker`
```

Comment:
This usually requires `sudo`

You could be fancy and do:
`sudo wget -O $(which docker) https://github.com/Metaswitch/calico-docker/releases/download/v0.5.2/docker`

...which means no need for the `chmod` or the `cp`.